terminologies/technique/singleCellRNASequencing.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+ "@id":"https://openminds.ebrains.eu/controlledTerms/technique/singleCellRNASequencing",

```

Comment:
change to "singleCellRNAsequencing" (cf. comments above about capitalization after abbreviations)